changelog.md

```diff
@@ -18,3 +18,6 @@
 
 ## 0.2.3
 - Add `ForEachAsync` extension for `IEnumerable`
+
+## 0.3.0
```

Comment:
You've specified a new release without changing the version in the `.csproj`

Reply:
dc15d640aff0a8f96fd9b452080620bce44e349e